taskfiles/install_darwin.yml

```diff
@@ -0,0 +1,61 @@
+version: '3'
+
+env:
+  PATH: "{{.PWD}}/bin:{{.PATH}}"
+
+vars:
+  HELM_VERSION: '{{.HELM_VERSION | default "3.9.4"}}'
+
+tasks:
+  brew-status:
+    interactive: true
+    cmds: 
+      - |
+        if which brew ; then
+          exit 0
+        else
+          echo -e "Brew is a prerequisite of this task. Pls refer to https://brew.sh/ for details"
+          exit 1
+        fi
+
+  ansible:
+      - brew install ansible
+
+  helm:
+    cmds:
+      - |
+        brew tap-new local/helm
+        brew extract --version={{.HELM_VERSION}} helm local/helm
+        brew install local/helm/helm@{{.HELM_VERSION}}
+
```

Comment:
@chriswessels unfortunately I can't find an easy way to install specific brew packages. smth like `brew install helm@3.9.4` doesn't work so the only workaround to install a specific version is the above (https://gist.github.com/ArgonQQ/cff4834dab6b254cc2140bb1454b47ef) but it's very very slow to run just for one package. So we can either leave latest as is or install binaries instead with `wget` - lmk pls